### ISSUE TITLE
Improvements for language filtering

### ISSUE THREAD

**Issue by Kimik0** [tootsuite/mastodon#3201]
The new language filtering in v1.4 seems to be working quite nicely ^_^

There are some aspects that could be improved though:
* There's no indication whatsoever on the main ("getting-started") page that a filter is active and you're missing toots.
* It's rather cumbersome to have to switch to the Preferences page, select some language(s), save, switch back, and find again where you left off, if you just want to temporarily filter out some things.

My suggestions:
* Most importantly, add a clear indication that a filter is active and the user is missing some toots. This could be done by showing the filtered toots in some collapsed form (a single line or something), or simply by an icon at the top of the column. #3085 requests the same thing for regexp filters on other columns.
* Make the language filtering accessible more quickly. One way would be to have a menu icon at the top of the column, much like the available filters on other columns, but a better way might be to have a "filter toots in `detected language`" item in the ••• menu under each toot.
* In addition to the first suggestion (indicate filter being active), have some indication of how much the user is missing by for example showing the number of toots filtered out from the past hour of timeline (or less for big instances).

* * * *
- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by Alienglyph** [comment]
We also need feedback/information on what language was detected in each toot.
> "filter toots in `detected language`" item in the ••• menu under each toot.

should do nicely for that.

**Comment by Cassolotl** [comment]
I've been keeping track of my toots since yesterday, noting the language codes in the atom feed. https://docs.google.com/spreadsheets/d/1BexKpvslEWedQSdhCH4Htlm0ZTxVbRjG449mgsBvs24/edit?usp=sharing

I'm wondering if tag-usernames are factored into language detection? I'm thinking that if they are, perhaps they shouldn't be.

**Comment by Swipe650** [comment]
The language filters don't seem to work for me at all on Mastodon.social and never have done. Both local and federated timelines are full of Arabic language toots even thought it is a filtered language.

**Comment by Swipe650** [comment]
Please could you add Persian to the list of filtered languages?

**Comment by Kimik0** [comment]
You mean Farsi? It's already on the list on the Settings page: "فارسی", between "Esperanto" and "Français".

I suppose you're referring to the recent new wave of toots in Farsi on mastodon.social? I'm not on that instance myself, so I can't say for certain, but the toots are correctly tagged as `language="fa"`, and if I pull a few of them to my local instance by looking up their urls, I can filter them out from the public TL too.

**Comment by KonomiKitten** [comment]
@Kimik0 might be broken only on Mastodon.social as I currently only have English selected to be shown and I see tons of Farsi on on Local Timeline so something is not working.

**Comment by Swipe650** [comment]
@KonomiKitten Yep, definitely broken on Mastodon.social. I'm not sure language filtering has ever worked properly on there.
